cnf-certification-test/accesscontrol/suite.go

```diff
@@ -185,16 +205,55 @@ var _ = ginkgo.Describe(common.AccessControlTestKey, func() {
 	})
 })
 
-// TestSecConCapabilities verifies that non compliant capabilities are not present
-func TestSecConCapabilities(env *provider.TestEnvironment) {
+func TestSysAdminCapability(env *provider.TestEnvironment) {
 	var badContainers []string
 	for _, cut := range env.Containers {
 		if cut.SecurityContext != nil && cut.SecurityContext.Capabilities != nil {
-			for _, ncc := range nonCompliantCapabilities {
-				if strings.Contains(cut.SecurityContext.Capabilities.String(), ncc) {
-					tnf.ClaimFilePrintf("Non compliant %s capability detected in container %s. All container caps: %s", ncc, cut.String(), cut.SecurityContext.Capabilities.String())
-					badContainers = append(badContainers, cut.String())
-				}
+			if strings.Contains(cut.SecurityContext.Capabilities.String(), "SYS_ADMIN") {
+				tnf.ClaimFilePrintf("Non compliant %s capability detected in container %s. All container caps: %s", "SYS_ADMIN", cut.String(), cut.SecurityContext.Capabilities.String())
+				badContainers = append(badContainers, cut.String())
+			}
+		}
+	}
+
+	testhelper.AddTestResultLog("Non-compliant", badContainers, tnf.ClaimFilePrintf, ginkgo.Fail)
+}
+
+func TestNetAdminCapability(env *provider.TestEnvironment) {
+	var badContainers []string
+	for _, cut := range env.Containers {
+		if cut.SecurityContext != nil && cut.SecurityContext.Capabilities != nil {
+			if strings.Contains(cut.SecurityContext.Capabilities.String(), "NET_ADMIN") {
+				tnf.ClaimFilePrintf("Non compliant %s capability detected in container %s. All container caps: %s", "NET_ADMIN", cut.String(), cut.SecurityContext.Capabilities.String())
```

Comment:
There is no need for `NET_ADMIN` string as a separate parameter.

Reply:
Fixed on all strings.